tier4_rtc_msgs/msg/Module.msg

```diff
@@ -3,14 +3,16 @@ uint8 type
 uint8 NONE = 0
 uint8 LANE_CHANGE_LEFT = 1
 uint8 LANE_CHANGE_RIGHT = 2
-uint8 AVOIDANCE_LEFT = 3
-uint8 AVOIDANCE_RIGHT = 4
-uint8 PULL_OVER = 5
-uint8 PULL_OUT = 6
-uint8 TRAFFIC_LIGHT = 7
-uint8 INTERSECTION = 8
-uint8 CROSSWALK = 9
-uint8 BLIND_SPOT = 10
-uint8 DETECTION_AREA = 11
-uint8 NO_STOPPING_AREA = 12
-uint8 OCCLUSION_SPOT = 13
+uint8 EXT_REQUEST_LANE_CHANGE_LEFT = 3
+uint8 EXT_REQUEST_LANE_CHANGE_RIGHT = 4
```

Comment:
@rej55 Actually if we add new module, I think it will be better if you add in the last row, so that the old module number will not be affected

Reply:
Fixed in https://github.com/tier4/tier4_autoware_msgs/pull/66/commits/5c501886773fa580ae9da0837de3b25780fd56a3